TrackPropagation/RungeKutta/test/rkTest_cfg.py

```diff
@@ -25,10 +25,12 @@
 
 #process.load("Configuration.StandardSequences.MagneticField_35T_cff")
 
-process.load("Configuration.StandardSequences.MagneticField_38T_cff")
+#process.load("Configuration.StandardSequences.MagneticField_38T_cff")
 
 #process.load("Configuration.StandardSequences.MagneticField_40T_cff")
 
+process.load("MagneticField.Engine.volumeBasedMagneticField_090322_2pi_scaled_cfi")
```

Comment:
do not understand "unsupported way of map loading": it works, so it is supported.